cps/uploader.py

```diff
@@ -320,22 +322,36 @@ def video_metadata(tmp_file_path, original_file_name, original_file_extension):
             identifiers=[])
         return meta
 
+def sanitize_path(path):
+    """Sanitize the file path to prevent command injection."""
+    return shlex.quote(path)
+
 def generate_video_cover(tmp_file_path):
     ffmpeg_executable = os.getenv('FFMPEG_PATH', 'ffmpeg')
-    ffmpeg_output_file = os.path.splitext(tmp_file_path)[0] + '.cover.jpg'
+    if not ffmpeg_executable:
+        log.error('FFMPEG_PATH environment variable is not set.')
+        return None
+
+    sanitized_input_path = sanitize_path(tmp_file_path)
+    output_file_path = os.path.splitext(tmp_file_path)[0] + '.cover.jpg'
+    sanitized_output_path = sanitize_path(output_file_path)
+
     ffmpeg_args = [
         ffmpeg_executable,
-        '-i', tmp_file_path,
-        '-vframes', '1',
-        '-y', ffmpeg_output_file
+        '-i', sanitized_input_path,
+        '-vf', 'fps=1,thumbnail,select=gt(scene\,0.1),scale=-1:720',
+        '-frames:v', '1',
+        '-vsync', 'vfr',
+        '-y',
+        sanitized_output_path
     ]
 
     try:
         ffmpeg_result = run(ffmpeg_args, capture_output=True, check=True)
```

Comment:
Let's be explicit:

```suggestion
        ffmpeg_result = subprocess.run(ffmpeg_args, capture_output=True, check=True)
```

...and get rid of the import on top.

Good enough to merge otherwise.